website/docs/docs/integrate/quickstart-semantic-layer.md

```diff
@@ -0,0 +1,292 @@
+---
+title: "Quickstart"
+id: quickstart-semantic-layer
+description: "Define metrics and set up the dbt Semantic Layer"
+sidebar_label: "Quickstart"
+---
+
+# dbt Semantic Layer quickstart
+
+:::info 📌
+
+The dbt Semantic Layer is currently available for public preview! With the dbt Semantic Layer, you'll be able to centrally define business metrics, remove code duplication and inconsistency, create self-service in downstream tools, and more! 
+
+
+For more info on public preview and plan availability, check out the [public preview](/docs/integrate/quickstart-semantic-layer#public-preview) and [product architecture](/docs/integrate/dbt-semantic-layer#product-architecture) section.
```

Comment:
Should say "Public Preview."

Can we also link the upcoming dbt product states doc from here?
https://docs.google.com/document/d/1zy8R4NdBCp3TPWEQh0srXukqj45Zt7wZWOn1CAIEl6E/edit